integration/test_fulltext.py

```diff
@@ -299,3 +300,370 @@ def test_custom_punctuation(self):
         # @ NOT configured as separator - should not be able with split words
         result = client.execute_command("FT.SEARCH", "idx", '@content:"test"')
         assert result[0] == 0
+
+    def test_ft_info_basic_text_index(self):
```

Comment:
We should think about how we structure our amztests at some point. I don't think isolating all our testing in fulltext specific files will always be the way to go. For example, we may want a test_ft_info.py file that contains tests for all ft.info fields. It makes more sense to me to group based on feature (the FT.INFO command) rather than index type in this case. Although if we can reduce the number of tests we're adding here, I think it's fine to leave just one or two in this file for now.